Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1314 @@
+        'Title': result.get('title'),
+        'Description': result.get('description'),
+        'Severity': result.get('severity'),
+        'Link': result.get('link'),
+        'Tags': result.get('tags'),
+        'Visibility': result.get('visibility')
+    }
+
+    context = {
+        'ID': result.get('id'),
+        'Timestamp': timestamp_to_datestring(result.get('timestamp')),
+        'Title': result.get('title'),
+        'Description': result.get('description'),
+        'Severity': result.get('severity'),
+        'Link': result.get('link'),
+        'Tags': result.get('tags'),
+        'IOCs': result.get('iocs_v2'),
+        'Visibility': result.get('visibility')
+    }
+
+    iocs = result.get('iocs_v2', [])
+    if iocs:
+        for ioc in iocs:
+            ioc_contents.append({
+                'ID': ioc.get('id'),
+                'Match_type': ioc.get('match_type'),
+                'Values': ioc.get('values'),
+                'Field': ioc.get('field'),
+                'Link': ioc.get('link')
+            })
+
+    readable_output = tableToMarkdown(f'Report "{report_id}" information', contents, headers, removeNull=True)
+    ioc_output = tableToMarkdown(f'The IOCs for the report', ioc_contents, removeNull=True)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Report',
+        outputs_key_field='id',
+        outputs=context,
+        readable_output=readable_output + ioc_output,
+        raw_response=result
+    )
+    return results
+
+
+def get_ignore_ioc_status_command(client: Client, args: Dict) -> str:
+    report_id = args.get('report_id')
+    ioc_id = args.get('ioc_id')
+
+    result = client.get_ignore_ioc_status_request(report_id, ioc_id)
+
+    if not result.get('ignored'):
+        return f'IOC {ioc_id} status is false'
+    else:
+        return f'IOC {ioc_id} status is true'
+
+
+def ignore_ioc_command(client: Client, args: Dict) -> str:
+
+    report_id = args.get('report_id')
+    ioc_id = args.get('ioc_id')
+
+    client.ignore_ioc_request(report_id, ioc_id)
+
+    return f'The IOC {ioc_id} for report {report_id} will not match future events for any watchlist.'
+
+
+def reactivate_ioc_command(client: Client, args: Dict) -> str:
+
+    report_id = args.get('report_id')
+    ioc_id = args.get('ioc_id')
+
+    client.reactivate_ioc_request(report_id, ioc_id)
+
+    return f'IOC {ioc_id} for report {report_id} will match future events for all watchlists.'
+
+
+def create_report_command(client: Client, args: Dict) -> CommandResults:
+
+    title = args.get('title')
+    description = args.get('description')
+    tags = argToList(args.get('tags'))
+    ipv4 = argToList(args.get('ipv4'))
+    ipv6 = argToList(args.get('ipv6'))
+    dns = argToList(args.get('dns'))
+    md5 = argToList(args.get('md5'))
+    ioc_query = argToList(args.get('ioc_query'))
+    severity = args.get('severity')
+    timestamp = date_to_timestamp(args.get('timestamp')) / 1000
```

Comment:
`timestamp` will be of type `float`, and `create_report_request` expects an `int`, so need to cast it to `int`